### PR TITLE
feat: Show Loader while logout (AR-2419)

### PR DESCRIPTION
**Pull request by borichellow**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-2419" title="AR-2419" target="_blank">BugAR-2419</a>  Show a loading indicator on log out
  </td></table>
  <br />
 

# What's new in this PR?

### Issues

Logging out takes some time but without any UI signal to the user, so the user could think it didn't work. 

### Solutions

Show an un-removable loader while logging out, so user can't do anything and inform him that "it's in progress"

### Attachments (Optional)

image

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-1661/1/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.1.1(6234219)(6234219)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.1.1(6234219)(6234219)-staging-compat.apk)

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -447,6 +447,7 @@
     <string name="user_profile_role_change_error">There was an error trying to change the role. Please check your internet connection and try again</string>
     <string name="user_profile_unblock_user">Unblock User</string>
     <string name="user_profile_group_remove_button">Remove from group</string>
+    <string name="user_profile_loggin_out_progress">Logging out...</string>
```
Comment (alexandreferris) [inline app/src/main/res/values/strings.xml]: small typo 🙈 
```suggestion
    <string name="user_profile_logging_out_progress">Logging out...</string>
```

**Review by ohassine — APPROVED** [review]
LGTM

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-1661/2/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.1.1(6241209)(6241209)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.1.1(6241209)(6241209)-staging-compat.apk)

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-1661/3/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.1.1(6241610)(6241610)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.1.1(6241610)(6241610)-staging-compat.apk)

**Review by alexandreferris — APPROVED** [review]

**Pull request merged**